Marlin/configuration_store.cpp

```diff
@@ -498,6 +503,10 @@ void Config_Postprocess() {
       EEPROM_READ(planner.min_travel_feedrate_mm_s);
       EEPROM_READ(planner.min_segment_time);
       EEPROM_READ(planner.max_jerk);
+
+      #if DISABLED(NO_WORKSPACE_OFFSETS)
+        float home_offset[XYZ];
+      #endif
       EEPROM_READ(home_offset);
```

Comment:
Doesn't compile:
`'home_offset' was not declared in this scope`.

I guess it's not needed with `NO_WORKSPACE_OFFSETS`, but we have to read/skip this EEPROM value in any case because the EEPROM layout will not fit for the following values if we continue reading without that.